dhis-2/dhis-support/dhis-support-db-migration/src/main/resources/org/hisp/dhis/db/migration/2.33/V2_33_38__Add_not_null_trackedentitytypeattribute_foreign_keys.sql

```diff
@@ -0,0 +1 @@
+ALTER TABLE trackedentitytypeattribute ALTER COLUMN trackedentityattributeid SET NOT NULL;
```

Comment:
You can club both the scripts into one file instead of having them as separate scripts.